tools/autograd/gen_python_functions.py

```diff
@@ -158,9 +158,13 @@
     "fill.Tensor",  # only used by the functionalization pass
     "fill.Scalar",  # only used by the functionalization pass
     "lift.*",
-    "normal_functional",  # only used by the functionalization pas
+    "normal_functional",  # only used by the functionalization pass
     "nbytes",
     "itemsize",
+    "batch_norm_with_update",
+    "batch_norm_with_update.out",
```

Comment:
You don't need to add variants here IIRC?

Reply:
Looks like I get the following error if I remove this:
```
RuntimeError: While identifying overloads, we found an out schema _batch_norm_with_update(Tensor input, Tensor? weight, Tensor? bias, Tensor running_mean, Tensor running_var, double momentum, double eps, *, TensorList[4]
 out=None) without a corresponding non-out variant. We expected the non-out variant to have schema:
- _batch_norm_with_update(Tensor input, Tensor? weight, Tensor? bias, Tensor running_mean, Tensor running_var, double momentum, double eps)
```